exec_test.go

```diff
@@ -529,6 +529,10 @@ print("Your text was", v)
 			ctx, cancel := context.WithTimeout(context.Background(), 20*time.Second)
 			defer cancel()
 
+			if tc.text == "" {
```

Comment:
nitpick: I would do it just after the ctx initialization.

This is not because changing the text value will eat a little bit of the 20s timeouts, but more than this is part of the "API" / code behaviour to execute the command itself. WDYT? (yes, this is really a personal preference, but as you wish)